docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -1,332 +1,407 @@
-
-ie = IECore()
-ie.get_metric(device_name=device,
-              metric_name="OPTIMIZATION_CAPABILITIES")
-```
-
-### Configuring the Individual Devices and Creating the Auto-Device on Top
-
-It is possible to configure each individual device as usual and create the "AUTO" device on top:
-
-```python
-from openvino.inference_engine import IECore
-
-ie = IECore()
-net = ie.read_network(model=path_to_model)
-
-cpu_config = {}
-gpu_config = {}
-
-ie.set_config(config=cpu_config, device_name="CPU")
-ie.set_config(config=gpu_config, device_name="GPU")
-
-# Load the network to the AUTO device
-exec_net = ie.load_network(network=net, device_name="AUTO")
-```
-
-Alternatively, you can combine all the individual device settings into single config file and load it, allowing the Auto-device plugin to parse and apply it to the right devices. See the code example here:
+For C++ API
+@sphinxdirective
+.. code-block:: sh
 
-```python
-from openvino.inference_engine import IECore
+   ov::runtime::Core::get_available_devices() (see Hello Query Device C++ Sample)
+@endsphinxdirective
 
-# Init the Inference Engine Core
-ie = IECore()
+For Python API
+@sphinxdirective
+.. code-block:: sh
 
-# Read a network in IR or ONNX format
-net = ie.read_network(model=path_to_model)
+   openvino.runtime.Core.available_devices (see Hello Query Device Python Sample)
+@endsphinxdirective
 
-full_config = {}
 
-# Load the network to the AUTO device
-exec_net = ie.load_network(network=net, device_name="AUTO", config=full_config)
-```
+### Performance Hints
+The ov::hint property enables you to specify a performance mode for the plugin to be more efficient for particular use cases.
 
-### Using the Auto-Device with OpenVINO Samples and Benchmark App
+#### ov::hint::PerformanceMode::THROUGHPUT
+This mode prioritizes high throughput, balancing between latency and power. It is best suited for tasks involving multiple jobs, like inference of video feeds or large numbers of images.
 
-Note that every OpenVINO sample or application that supports the "-d" (which stands for "device") command-line option transparently accepts the Auto-device. The Benchmark Application is the best example of the optimal usage of the Auto-device. You do not need to set the number of requests and CPU threads, as the application provides optimal out-of-the-box performance. Below is the example command-line to evaluate AUTO performance with that:
+#### ov::hint::PerformanceMode::LATENCY
+This mode prioritizes low latency, providing short response time for each inference job. It performs best for tasks where inference is required for a single input image, like a medical analysis of an ultrasound scan image. It also fits the tasks of real-time or nearly real-time applications, such as an industrial robot's response to actions in its environment or obstacle avoidance for autonomous vehicles.
+Note that currently the ov::hint property is not supported by VPUX and Myriad devices.
```

Comment:
What are "VPUX and Myriad devices" ?

Reply:
I would say these are the devices covered by the "device_name" parameter as Myriad and VPUX. It follows the convention laid out in the table at the top of the article. I guess we could use the full names of products, but, apparently, VPUX may include more than one product. Unlike Myriad. In this context we are referring to parameter values rather than devices themselves, I think.

Reply:
Note that currently the ov::hint property is supported by CPU and GPU plugins. @kblaszczak-intel  @andrew-zaytsev Can we update as this to avoid confusing?

Reply:
@peterchen-intel Done and done ;) it even sounds better :)